src/transceivercontroller/DefaultTransceiverController.ts

```diff
@@ -586,13 +586,10 @@ export default class DefaultTransceiverController
   }
 
   /**
-   * Adds a transceiver to the peer connection and performs additional necessary setup.
+   * Set up and add a video transceiver to the peer connection.
    */
-  private addTransceiver(
-    trackOrKind: string | MediaStreamTrack,
-    init?: RTCRtpTransceiverInit
-  ): RTCRtpTransceiver {
-    const transceiver = this.peer.addTransceiver(trackOrKind, init);
+  protected setupVideoTransceiver(init?: RTCRtpTransceiverInit): RTCRtpTransceiver {
```

Comment:
- Only optional arguments are not recommended, from earlier method do we need to keep `trackOrKind` around?
- We currently only explicilty want to pass 'video' so instead of creating a new method, should we use `addTransceiver` and pass 'video' to it? Not sure if this is being re-used elsewhere in this class and whether some string or media stream track is being passed.

Reply:
I thought that the capability to add an existing MediaStreamTrack (rather than having the PeerConnection create a new one) is important. But will need to check if that is being used.

Reply:
This function was actually added by the RED changes to setup passthrough transforms for video transceivers. Otherwise, RED would cause video send/receive to not work. `trackOrKind` is not needed if this is only intended to be used for video transceivers and I've updated all other uses of this function to not pass a string/media stream track in this PR.

Reply:
Thinking more about this we would probably want to keep the ability to add a `MediaStreamTrack`, I will undo the refactors for this function as it is not necessary for the fix.

Reply:
I reverted these refactors in latest revision.